backend/api/Areas/Tools/Helpers/ImportPropertiesHelper.cs

```diff
@@ -46,6 +46,82 @@ public ImportPropertiesHelper(IPimsAdminService pimsAdminService, ILogger logger
         #endregion
 
         #region Methods
+        /// <summary>
+        /// Parse the value to extract the PID number.
+        /// If it's invalid it will return -1.
+        /// </summary>
+        /// <param name="value"></param>
+        /// <returns></returns>
+        private int ParsePid(string value)
+        {
+            if (int.TryParse(value?.Replace("-", ""), out int pid))
+                return pid;
+
+            return -1;
+        }
+
+        /// <summary>
+        /// Delete properties from inventory that match the parcel PID or building name.
+        /// </summary>
+        /// <param name="properties"></param>
+        /// <param name="updatedBefore">Only allow deletes to properties updated before this date.</param>
```

Comment:
should the updatedBefore also apply to the createdOn date in the DB? we have both fields, but only createdOn is populated when a row is initially created.